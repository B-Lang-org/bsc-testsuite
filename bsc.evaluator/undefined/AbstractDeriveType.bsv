(* synthesize *)
module mkAbstractDeriveType();
   Type t = ?;
   messageM(printType(t));
endmodule
