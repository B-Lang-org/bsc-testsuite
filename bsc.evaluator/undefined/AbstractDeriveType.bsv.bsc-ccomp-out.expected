checking package dependencies
compiling AbstractDeriveType.bsv
code generation for mkAbstractDeriveType starts
Error: "AbstractDeriveType.bsv", line 3, column 13: (S0015)
  Bluespec evaluation-time error: Attempt to use undetermined Type
  During elaboration of `messageM' at "AbstractDeriveType.bsv", line 4, column
  4.
  During elaboration of `mkAbstractDeriveType' at "AbstractDeriveType.bsv",
  line 2, column 8.
